source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -289,9 +305,25 @@
                                     return valueItem.value() === "" || (valueItem.value() >= 1 && valueItem.value() <= 4);
                                 })
                                 self.isDirty.register(valueItem.value);
+                                valueItem.showComments = false
```

Comment:
Is it the same "showComments" that [appears in line 323](https://github.com/CommonJobs/CommonJobs/pull/479/files#diff-1b28b31e2207786095b5affe916c595bR323)?

In that case, notice that one of them is a _boolean_ and the other one is an _Obserbable<boolean>_.